app/routes/app/AppRoute.js

```diff
@@ -141,6 +142,12 @@ class App extends PureComponent<AppProps> {
 
           <PhotoUploadStatus />
 
+          <CookieConsent
+            content="Når du besøker abakus.no installerer vi cookies som er nødvendig for at du skal kunne benytte deg av nettsidene våre."
```

Comment:
```suggestion
            content="Når du besøker abakus.no, installerer vi cookies som er nødvendig for at du skal kunne benytte deg av nettsidene våre."
```

Reply:
Thanks for the review. I will correct the grammar, but it is not easy to use css-variables as the CookieConset component needs inline styling, so it's not possible to use css classes.

Reply:
Ait, no worries :+1: